pages/welcome/sections/general_section.py

```diff
@@ -0,0 +1,105 @@
+from dash import dcc, html
+import dash
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+layout = html.Div(
+    className="tab_section_container",
+    children=[
+        html.Div(
+            className="card_section_container",
+            children=[
+                html.Div(
+                    className="card_section_description",
+                    children=[
+                        html.H1("8Knot Pages"),
+                        html.P(
+                            """
```

Comment:
Some Rewording here. What message are you trying to get across here? Maybe something like 
"Each page gives context on a community from a different perspective"

Reply:
++ bet